status-messages/cloudformation.yml

```
@@ -0,0 +1,64 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  HyP3URL:
+    Type: String
+
+  QueueName:
+    Type: String
+
+  MattermostPAT:
+    Type: String
+
+Resources:
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "/aws/lambda/${Lambda}"
+
+  Role:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          Action: sts:AssumeRole
+          Principal:
+            Service: lambda.amazonaws.com
+          Effect: Allow
+      Policies:
+        - PolicyName: policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action: sqs:GetQueueAttributes
+                Resource: !Sub "arn:aws:sqs:${AWS::Region}:${AWS::AccountId}:${QueueName}"
+
+  Lambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Environment:
+        Variables:
+          HYP3_URL: !Ref HyP3URL
+          MATTERMOST_PAT: !Ref MattermostPAT
+      Code: src/
+      Handler: status_messages.lambda_handler
+      Role: !GetAtt Role.Arn
+      Runtime: python3.12
```

Apparently the default timeout is 3 seconds according to the [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-timeout). That's probably enough given that you said it runs in like a second, but maybe we should increase to 10 seconds just to account for random network lag when making `boto3` calls, etc?
```suggestion
      Runtime: python3.12
      Timeout: 10
```